src/app/pcasts/models/user.py

```diff
@@ -17,6 +18,7 @@ class User(Base):
 
   def __init__(self, **kwargs):
     self.google_id = kwargs.get('google_id')
+    self.facebook_id = kwargs.get('facebook_id')
```

Comment:
B/c google_id can be null, we gotta change the default username on not providing one.  If you always specify one in the DAO, remove the default one so the creation of the model fails (`username` would be set to `None`, which is illegal).